infra/littlechef.cfg-example

```diff
@@ -0,0 +1,12 @@
+[userinfo]
+user = root
+password =
+#keypair-file = 
+ssh-config = ~/.ssh/config
+#encrypted_data_bag_secret = 
+[kitchen]
+node_work_path = /tmp/chef-solo/
+
+[kitchen]
+berksfile = Berksfile
+berksfile_cookbooks_directory = berks-cookbooks
```

Comment:
you MUST copy this and have a littlechef.cfg, just copying should be enough.
Sometimes you may have passwords here etc, but we're all ssh key driven